src/MSIdentityScaffolding/Microsoft.DotNet.MSIdentity/Tool/ConsoleLogger.cs

```diff
@@ -0,0 +1,61 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.DotNet.MSIdentity.Tool
+{
+    internal class ConsoleLogger : IConsoleLogger
+    {
+        private bool _jsonOutput;
+
+        public ConsoleLogger(bool jsonOutput = false)
+        {
+            _jsonOutput = jsonOutput;
+            Console.OutputEncoding = Encoding.UTF8;
```

Comment:
Have you verified this works in the console in Windows with non-ASCII characters?

Reply:
It works with no issues in Windows console. The azure portal even handles non-ascii characters.